dart/pubspec.yaml

```diff
@@ -1,20 +1,13 @@
 name: flat_buffers
-version: 1.12.0
-description: >
-  FlatBuffers reading and writing library for Dart.  Use the flatc compiler to
-  generate Dart classes for a FlatBuffers schema, and this library to assist with
-  reading and writing the binary format.
-
-  Based on original work by Konstantin Scheglov and Paul Berry of the Dart SDK team.
-authors:
-- Dan Field <dfield@gmail.com>
-- Konstantin Scheglov
-- Paul Berry
+version: 1.12.0+1
+description:
+  Library for reading and writing FlatBuffers. Use the flatc compiler to
+  generate Dart classes from a FlatBuffer schema, and this library read and write the binary format.
 homepage: https://github.com/google/flatbuffers
 documentation: https://google.github.io/flatbuffers/index.html
 dev_dependencies:
   test: ^1.3.0
   test_reflective_loader: ^0.1.4
   path: ^1.5.1
 environment:
-  sdk: '>=2.0.0-dev.28.0 <3.0.0'
\ No newline at end of file
+  sdk: '>=2.0.0 <3.0.0'
```

Comment:
new line at EOF.

Given this constraint, we should also remove the line about supporting Dart 1.x from the README.

Reply:
Sure.
I can't find any reference to Dart 1.x in the `README.md`. Do you mean the comments regarding Dart 1.x support in `CHANGELOG.md`?

Reply:
Ahh, sorry, it was changelog. Nevermind then. Just the newline :)